ircb/cli/network.py

```diff
@@ -28,8 +28,8 @@ def network_cli():
               type=click.Choice(
                   ['CERT_NONE', 'CERT_OPTIONAL', 'CERT_REQUIRED']))
 @coroutinize
-def create(user, network_name, host, port, nick, realname, username, password,
-           usermode,ssl,ssl_verify):
+def network_create(user, network_name, host, port, nick, realname, username, password,
```

Comment:
@rtnpro is this the create() that you were looking to rename? What was the reason behind the name change?